crates/context/config/src/client/env/config/mutate.rs

```diff
@@ -162,6 +170,39 @@ impl<'a> Method<Icp> for Mutate<'a> {
     }
 }
 
+impl<'a> Method<Stellar> for Mutate<'a> {
+    type Returns = ();
+    const METHOD: &'static str = "mutate";
+
+    fn encode(self) -> eyre::Result<Vec<u8>> {
+        let env = Env::default();
+        let signer_sk = SigningKey::from_bytes(&self.signing_key);
+
+        let signer_id: [u8; 32] = signer_sk.verifying_key().rt()?;
```

Comment:
for these kinds of cases:

```rust
let some_var: [u8; 32] = thing.rt().expect("infallible conversion");
```

you can call `.as_bytes` instead:

```rust
let some_var: [u8; 32] = thing.as_bytes();
```

no need for error handling, it's an infallible operation